### PR TITLE
fix: added table_id to RegionStat

### PR DESCRIPTION
**Pull request by tarunsamanta2k20**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)
linked issue of https://github.com/GreptimeTeam/greptimedb/issues/1566
## What's changed and what's your intention?

_PLEASE DO NOT LEAVE THIS EMPTY !!!_

Please explain IN DETAIL what the changes are in this PR and why they are needed:

- Summarize your change (**mandatory**)
- How does this PR work? Need a brief introduction for the changed logic (optional)
- Describe clearly one logical change and avoid lazy messages (optional)
- Describe any limitations of the current code (optional)

## Checklist

- [ ]  I have written the necessary comments.
- [ ]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

File: proto/greptime/v1/meta/common.proto [hunk 1]
```diff
@@ -39,6 +39,7 @@ message TableName {
   string catalog_name = 1;
   string schema_name = 2;
   string table_name = 3;
+  uint64 table_id= 4;
```
Comment (MichaelScofield) [inline proto/greptime/v1/meta/common.proto:42]: Why define `table_id` here since it's already in `RegionStat`?

Comment (tarunsamanta2k20) [reply proto/greptime/v1/meta/common.proto:42]: Got it.

**Comment by MichaelScofield** [comment]
@tarunsamanta2k20 can you generate the proto's go code? The CI failed. You can just run `make` under the project's root path.

**Comment by MichaelScofield** [comment]
@tarunsamanta2k20 yes, execute `make` and the go protobuf code will be generated

**Comment by tarunsamanta2k20** [comment]
@MichaelScofield then what do I have to do?

**Comment by tarunsamanta2k20** [comment]
@MichaelScofield thread 'main' panicked at 'Could not find `protoc` installation and this build crate cannot proceed without

**Comment by MichaelScofield** [comment]
@tarunsamanta2k20 After executing "make" command, the go protobuf source codes should be generated, then you can commit & push them as well. Note that generating protobuf source codes requires pulling some docker image, so make sure you have docker daemon running in your machine.

**Comment by tarunsamanta2k20** [comment]
@MichaelScofield Done.

**Review by killme2008 — APPROVED** [review]
LGTM

**Review by MichaelScofield — APPROVED** [review]

**Pull request merged**